app/test_main.py

```diff
@@ -1,3 +1,60 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,human_years",
```

Comment:
You can replace it with one tuple of ages, and then unpack it inside your function in test as arguments